table/server/master/src/main/java/alluxio/master/table/TableMaster.java

```diff
@@ -37,6 +37,15 @@
   boolean attachDatabase(String dbName, String dbType, Map<String, String> configuration)
       throws IOException;
 
+  /**
+   * Remove an existing database in the catalog.
+   *
+   * @param dbName the database name to remove
+   * @return true if deletion is successful
+   */
+  boolean detachDatabase(String dbName)
```

Comment:
Is the database name a unique identifier?

Reply:
When attaching, the database name must be unique in the Alluxio master's catalog namespace - it's an entry in the `mDBs` map in `AlluxioCatalog.java`